extensions/amp-analytics/0.1/vendors/deepbi.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {jsonLiteral} from '../../../../src/json';
+
+const DEEPBI_CONFIG = jsonLiteral({
+  'requestOrigin':'https://${endpoint}/v1/pixel',
+  'vars': {
+    'gdpr': true,
+    'endpoint': 'api.deep.bi',
+  },
```

Comment:
`/v1/pixel` needs to be moved from the end of `requestOrigin` to the beggining of `base`, or else it will be ignored.